copyright_updater.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2018-2022 UT Longhorn Racing Solar
+# Python script to change copyright information
+import os
+from pathlib import Path
+directory = r"/home/raydog/BPS" #replace raydog with your user name to replace on local
+headers = ['/* Copyright (c) 2018-2022 UT Longhorn Racing Solar */\n', '# Copyright (c) 2018-2022 UT Longhorn Racing Solar\n']
+
+
+for root, subdirectories, files in os.walk(directory):
+    for file in files:
+        if file.endswith(".c") or file.endswith('.h') or file.endswith('.py'):  #finds all .c .h .py files
+            filepath = os.path.join(root, file)
+            f = open(filepath,"r",encoding = "ISO-8859-1")  #meant to fix bug with not recognizing byte 0x92
+            text = f.readlines()
```

Comment:
close the file before here before the write operation

Reply:
shouldn't be an issue, file is closed at end anyways